SteamBot/Bot.cs

```diff
@@ -713,13 +713,15 @@ void HandleSteamMessage(ICallbackMsg msg)
 
             msg.Handle<SteamClient.DisconnectedCallback> (callback =>
             {
+                var mobileAuthCode = GetMobileAuthCode();
                 if(IsLoggedIn)
                 {
                     IsLoggedIn = false;
                     CloseTrade();
                     Log.Warn("Disconnected from Steam Network!");
-                }
-
+				} else if(string.IsNullOrEmpty(mobileAuthCode)) {						
+					Thread.Sleep(60 * 1000);
```

Comment:
Sleeping the entire thread is probably a bad idea, please `await Task.Wait()` instead